vm-to-vm-bandwidth-meter/azuredeploy.json

```diff
@@ -0,0 +1,611 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "probeVmSize": {
```

Comment:
We're just trying to provide good dev samples, if someone wants to try A8 instead of A2 it's pretty easy to update the list.  It might be worth adding a link to this: https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-sizes/ so folks can understand the differences